hw_decorator.py

```diff
@@ -0,0 +1,99 @@
+import functools
+
+
+# ЗАДАЧА-1
+# Написать свой декоратор который будет проверять остаток от деления числа 100 на результат работы функции ниже.
+# Если остаток от деления = 0, вывести сообщение "We are OK!», иначе «Bad news guys, we got {}» остаток от деления.
+
+def divide_by_hundred(my_func):
+    """The decorator function which checks input for a valid type (int or float) and then checks if the result
+        of my_func is divided by 100 without rest or not."""
+    @functools.wraps(my_func)
+    def is_hundred_div(arg_one, arg_two):
+        if not isinstance(arg_one, int) or not isinstance(arg_one, int): #or
+            if not isinstance(arg_one, float) or not isinstance(arg_one, float):
+                raise ValueError("Only integer and float types are supported.")
+        result =  my_func(arg_one, arg_two) % 100
+        if not result:
```

Comment:
I would make it more re-usable like:
    def is_hundred_div(*args):
        for argument in args:
            if not isinstance(argument, (int, float)):
                raise ValueError("Only integer and float types are supported.")
        result = my_func(*args) % 100